cypress/e2e/shared/registration.cy.ts

```diff
@@ -68,30 +68,18 @@ export function shouldBehaveLikeARegistrationForm(registrationType: Registration
       }
     );
 
-    it('requires Replicate Number to have number part as an an integer', () => {
+    it('requires Replicate Number to be an alphanumeric string of maximum length 7 ', () => {
       cy.findByTestId('Replicate Number').type('1.1').blur();
       checkReplicateWarningIsVisible();
-    });
 
-    it('requires Replicate Number to have number part greater than 0', () => {
-      cy.findByTestId('Replicate Number').clear().type('-1').blur();
+      cy.findByTestId('Replicate Number').clear().type('1ab-').blur();
       checkReplicateWarningIsVisible();
-    });
 
-    it('requires Replicate Number to have number part  greater than or equal to 1', () => {
-      cy.findByTestId('Replicate Number').type('0').blur();
-      checkReplicateWarningIsVisible();
-    });
-    it('requires Replicate Number to be number or number followed by a lowercase letter', () => {
-      cy.findByTestId('Replicate Number').type('0ab').blur();
-      checkReplicateWarningIsVisible();
-    });
-    it('requires Replicate Number to be number or number followed by a lowercase letter', () => {
-      cy.findByTestId('Replicate Number').type('5ab').blur();
+      cy.findByTestId('Replicate Number').clear().type('1ab6789').blur();
```

Comment:
Why should a warning be shown for the replicate value "1ab6789"?

Reply:
That's a mistake, thought it was >7. Chnaged it now. But, I wonder why it worked. Anyway, rechecked it to make sure it validates the correct rule and it all seems to work